src/main/java/loginPage/Homepage.java

```diff
@@ -0,0 +1,41 @@
+package loginPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+public class Homepage {
+    WebDriver driver;
+    public Homepage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver,this);
+    }
+    @FindBy(xpath = "//input[@id=\"user-name\"]")
+    WebElement username;
+
+    @FindBy(xpath = "//input[@id=\"password\"]") WebElement password;
+
+    @FindBy(xpath = "//input[@id=\"login-button\"]") WebElement button;
+
+    @FindBy(xpath = "//h3[@data-test = \"error\"]") WebElement error;
+
+    public void enterUsername(String user) {
```

Comment:
this is correct but in a more professional environment, you would want to create getters for you WebElements that way those getters would be exposed to anything you are using the page object while keeping your actual WebElements private.